x/nft/helper.go

```diff
@@ -27,19 +27,20 @@ func (*Validation) IsValidTokenID(id []byte) bool {
 	return len(id) >= minIDLength && len(id) <= maxIDLength
 }
 
-func FindActor(auth x.Authenticator, ctx weave.Context, t BaseNFT, action string) weave.Address {
+func FindActor(height int64, auth x.Authenticator, ctx weave.Context, t BaseNFT, action string) weave.Address {
```

Comment:
I would put height next to action for better readability and it would make more sense logically.